modules/gitops-verifying-argo-cd-self-healing-behavior.adoc

```diff
@@ -0,0 +1,32 @@
+[id="verifying-argo-cd-self-healing-behavior_{context}"]
+= Verifying Argo CD self-healing behavior
+
+Argo CD constantly monitors the state of deployed applications, detects differences between the specified manifests in Git and live changes in cluster, and automatically corrects them. This behavior is referred to as self-healing. 
+
+You can test and observe the self-healing behavior in Argo CD. 
+
+.Prerequisites
+
+* The sample application was deployed and configured.
+
+.Procedure
+
+. In the Argo CD dashboard, verify that your application has the `Synced` status.
+
+. Click the `app-spring-petclinic` tile in the Argo CD dashboard to view the application resources that are deployed to the cluster.
```

Comment:
If we use the `app-spring-petclinic` tile we will need to ensure the application created was named as `app-spring-petclinic`. So we will need to add it to the prerequisite. 
`synchronizing-your-application-application-with-your-git-repository.adoc` will need this prereq too. Unfortunately without links since these are modules.